plugins/arDominionB5Plugin/js/tooltip.js

```diff
@@ -1,5 +1,17 @@
-import Tooltip from "bootstrap/js/dist/tooltip";
-
-document
-  .querySelectorAll("[data-bs-toggle=tooltip]")
-  .forEach((element) => new Tooltip(element));
+document.addEventListener("DOMContentLoaded", function () {
+  if (
+    window.location.pathname.endsWith("/add") ||
+    window.location.pathname.endsWith("/edit")
+  ) {
+    var elements = document.querySelectorAll('[id$="-help"]');
+    elements.forEach(function (idElement) {
+      idElement.style.display = "none";
+      parentElement = idElement.previousElementSibling;
+      if (parentElement != null) {
+        parentElement.setAttribute("data-bs-toggle", "tooltip");
+        parentElement.setAttribute("title", idElement.textContent.trim());
+        var tooltip = new bootstrap.Tooltip(parentElement);
+      }
+    });
+  }
+});
```

Comment:
I gave it a try and this is the `tooltip.js` file I ended with, using jQuery:

```js
import Tooltip from "bootstrap/js/dist/tooltip";

(($) => {
  "use strict";

  $(() => {
    $("body.edit.show-edit-tooltips [id$=-help]").each((_, el) => {
      $(el)
        .prevAll(":not([type=hidden]):first")
        .attr({ "data-bs-toggle": "tooltip", title: el.textContent.trim() });
    });

    $("[data-bs-toggle=tooltip]").each((_, el) => new Tooltip(el));
  });
})(jQuery);
```

Some notes:
- The first function (`(($) => {`) isolates this code from the global scope.
- The second one (`$(() => {`) makes sure the code is executed when the DOM has been loaded.
- The selector `body.edit.show-edit-tooltips [id$=-help]` checks that we are on an edit page with the show tooltips setting enabled.
- `prevAll(":not([type=hidden]):first")` gets the first non hidden previous sibling (for the autocompletes issue).
- `attr()` sets multiple attributes at the same time.
- Finally, we initialize all tooltips on the page.